CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [2.0.56]
```

Comment:
Might want to leave it in the Unreleased block and let the maintainers update a release version separately...

Reply:
No worries, I'll fix before release.